routingv8/request.go

```diff
@@ -501,7 +502,28 @@ func (t *SpanAttribute) String() string {
 		return string(SpanAttributeNames)
 	case SpanAttributeMaxSpeed:
 		return string(SpanAttributeMaxSpeed)
+	case SpanAttributeLength:
+		return string(SpanAttributeLength)
 	default:
 		return invalid
 	}
 }
+
+type RouteImportRequest struct {
+	TransportMode TransportMode
+	// Which attributes to return in the response.
```

Comment:
Follow Go conventions and have the name being documented as the first word (or on the first line).

https://go.dev/blog/godoc

> The convention is simple: to document a type, variable, constant, function, or even a package, write a regular comment directly preceding its declaration, with no intervening blank line. Godoc will then present that comment as text alongside the item it documents. For example, this is the documentation for the fmt package’s [Fprint](https://go.dev/pkg/fmt/#Fprint) function:

```go
// Fprint formats using the default formats for its operands and writes to w.
// Spaces are added between operands when neither is a string.
// It returns the number of bytes written and any write error encountered.
func Fprint(w io.Writer, a ...interface{}) (n int, err error) {
```

> Notice this comment is a complete sentence that begins with the name of the element it describes. This important convention allows us to generate documentation in a variety of formats, from plain text to HTML to UNIX man pages, and makes it read better when tools truncate it for brevity, such as when they extract the first line or sentence.

Reply:
Fixed